app/views/locker_applications/_index_view.html.erb

```diff
@@ -20,6 +20,9 @@
     <grid-item columns="lg-2 md-4 sm-12">
       <%= link_to 'Edit', edit_locker_application_path(locker_application) %>
       <%= link_to 'Assign Locker', assign_locker_application_path(locker_application) %>
+      <% if current_page?(awaiting_assignment_locker_applications_path) %>
```

Comment:
Is the method `put` needed if it's in the routes file?

Reply:
Possibly not necessary, retaining for clarity.